src/main/java/be/minelabs/recipe/molecules/Molecule.java

```diff
@@ -5,13 +5,19 @@
 public class Molecule extends PartialMolecule {
     private final Item item;
 
-    public Molecule(MoleculeGraph structure, Item item) {
+    private final Integer count;
+
+    public Molecule(MoleculeGraph structure, Item item, Integer count) {
         super(structure);
         this.item = item;
+        this.count = count;
```

Comment:
What is the use case for this count?

Reply:
Seems like it's only part of the recipes at first sight. Then the recipe might be a better way to store it? I.e. immediately as a result itemstack.

Reply:
If we want more items as an output. 
If an recipe gives 2 items then the count is 2.

Reply:
Do you also think it would fit better in the `MoleculeRecipe` then?